output/pixel_pusher.c

```diff
@@ -0,0 +1,272 @@
+#include "pixel_pusher.h"
+
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <SDL2/SDL.h>
+#include <sys/socket.h>
+#include <sys/time.h>
+#include <time.h>
+#include <unistd.h>
+
+#include "output/config.h"
+#include "util/err.h"
+
+// This file implements a PixelPusher output.  It is designed to discover only 1 PixelPusher,
+// but to use multiple grids attached to that PixelPusher
+
+static struct pp_discovery_packet * pp_info = NULL;
```

Comment:
Yeah this was a misguided attempt to save every possible byte of memory when the pixel pusher isn't in use.  Given that you can just compile radiance without pixel pusher support if you really care about it I'll just switch this to static allocation.